plugins/storage/pyparts/ceph_daemon_checks.py

```diff
@@ -97,6 +96,29 @@ def check_osd_msgr_protocol_versions(self):
             msg = ("{} OSDs do not bind to v2 address".format(len(v1_osds)))
             issue_utils.add_issue(issue_types.CephOSDWarning(msg))
 
+    def check_ceph_bluefs_size(self):
+        """
+        Check if the BlueFS metadata size is too large
+        """
+        bad_meta_osds = []
+        ceph_osd_df_tree = self.cli.ceph_osd_df_tree_json_decoded()
+        for device in ceph_osd_df_tree['nodes']:
```

Comment:
Now that we always expect to have the data, this would fail for an empty list as we have decided not to catch exceptions which would otherwise hide buggy code or missing data in sosreport, etc.

Reply:
Raised https://github.com/canonical/hotsos/issues/157